### PR TITLE
[MM 25656] Adds config subpath command

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
This PR adds the `mmctl config subpath` subcommand, that rewrites the assets in a given path to include a given subpath into account. This command doesn't require a client to run, as it only acts on a directory local to the tool.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-25656

#### Related Pull Requests
- [Has server changes](https://github.com/mattermost/mattermost-server/pull/15020)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,10 +5,10 @@ go 1.13
 require (
 	github.com/golang/mock v1.3.1
 	github.com/magefile/mage v1.9.0
-	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200627210001-eff2209a7e37
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200714073806-915a31cd8a65
```
Comment (mgdelacroix) [inline go.mod:8]: This module version is temporal for the tests to run correctly. It will change after the server PR gets merged

**Review by isacikgoz — APPROVED** [review]
Nice 👍

**Review by ethervoid — APPROVED** [review]
LGTM!

**Review by lindy65 — APPROVED** [review]
Thanks @mgdelacroix - this will be tested after merging 👍

**Pull request merged**